content/docs/quickstart/_index.md

```diff
@@ -128,6 +128,8 @@ curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v4.4
 curl -SL https://raw.githubusercontent.com/keptn/examples/master/quickstart/expose-keptn.sh | bash
 ```
 
+5. **Access Bridge**: you can now access the Keptn Web UI at `http://127.0.0.1.nip.io:8082/bridge`. For different way on how to expose your Keptn installation, please refer to <https://keptn.sh/docs/0.13.x/operate/install/#install-keptn>.
```

Comment:
For some reason, it shows a `1.` instead of `5.` in the preview. Can you get this fixed please:

![image](https://user-images.githubusercontent.com/729071/157411110-ef8234ce-5778-49b8-9972-b1aec10adf96.png)